src/api/app/models/obs_factory/staging_project.rb

```diff
@@ -207,11 +206,25 @@ def attributes
     end
 
     def build_state
-      return :building if building_repositories.present? || disabled_repositories.present?
+      return :building if building_repositories.present?
       return :failed if broken_packages.present?
       :acceptable
     end
 
+    def checks
+      # for stagings we ignore 'old' checks in disabled repositories
+      repos = disabled_repositories
+      project.checks.to_a.each do |check|
+        next unless repos.include?(check.status_report.checkable)
+        # we can't simply ignore them as missing_checks is calculated
+        # independently, so we would be left without missing checks
+        # but without checks - so better mark them as 'old/pending',
+        # but only in memory, no in the database.
+        check.state = :pending
```

Comment:
Doesn't this just mean that somewhere in our code we want to ignore the state? :thinking: I could imagine that changing it here will lead to unexpected side effects at some point in time, because noone would expect this.

Reply:
Yeah, perhaps introduce an explicit state instead? Make it 'disabled', but keep the stagin project in :testing?

Reply:
I think I don't know enough about the actual problem yet to answer this.
But it might be one solution. Another option might be to move the filter you use here out of this method and apply it where / when we need it.

Reply:
the whole status api is a little crude and leads to broken expectations - so I wouldn't want to extend this further. But you might have noticed that I didn't touch the staging namespace - as we need to reevaluate this, but on a larger scale. This works 'good enough' for now - as those working with the staging projects know what to look for.

Reply:
Alright, let's see when we bump into this^^